man/org.freedesktop.systemd1.xml

```diff
@@ -1787,6 +1791,9 @@ node /org/freedesktop/systemd1 {
       <para><varname>UnitPath</varname> encodes the currently active unit file search path. It is an array of
       file system paths encoded as strings.</para>
 
+      <para><varname>SoftRebootsCount</varname> encodes how many soft-reboots were successfully completed
+      since the last full boot.</para>
```

Comment:
should mention what the value of this is in the first cycle.

i.e. that this is 0 on the first cycle after switch root